### PR TITLE
Add new demos to README

### PR DESCRIPTION
**Pull request by slchow**

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@slchow, thanks for the pull request! Maintainers, we have a signed CLA from @slchow, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

File: README.md [hunk 1]
```diff
@@ -49,6 +49,10 @@ We appreciate attribution by including the Cesium logo and link in your app.
 ### Demos ###
 
 <p align="center">
+<a href="http://cesiumjs.org/demos/fishing/"><img src="http://cesiumjs.org/images/demos/fishing.jpg" height="150" /></a>&nbsp;
+<a href="http://cesiumjs.org/demos/moontrek/"><img src="http://cesiumjs.org/images/demos/moontrek.jpg" height="150" /></a>&nbsp;
+<a href="http://cesiumjs.org/demos/maven/"><img src="http://cesiumjs.org/images/demos/maven.jpg" height="150" /></a>&nbsp;
+<a href="http://cesiumjs.org/demos/spedmo/"><img src="http://cesiumjs.org/images/demos/spedmo.jpg" height="150" /></a>&nbsp;
```
Comment (emackey) [inline README.md]: These images appear to be broken links.  Is this PR waiting on a corresponding PR to the website?

**Comment by slchow** [comment]
Thanks @emackey 

I just had the paths wrong: it should be OK now.

**Comment by emackey** [comment]
These paths look OK now.  There are a few remaining broken ones, should we address them in this PR too?

https://cesiumjs.org/demos/images/RikiTraki.png
https://cesiumjs.org/demos/images/CubeGlobe.jpg
http://cesiumjs.org/demos/images/RedBull1.png
http://cesiumjs.org/demos/images/fodar/fodar_03_md.png
http://cesiumjs.org/images/CesiumViewer.png
http://cesiumjs.org/images/Sandcastle.png
http://cesiumjs.org/demos/images/PearlHarbor.png

**Comment by emackey** [comment]
No more `404` errors in my network tab 🎉 

Thanks @slchow!

**Pull request merged**